openapi.yaml

```diff
@@ -525,343 +605,340 @@ parameters:
-        example: '2018-12-31'
+        example: 1971-2-21
       contractType:
         type: string
-        description: Contract type code/ID.
-      locationID:
+        enum: [P, S, O]
+        description: |
+          Contract type code/Id.
+          `P` - Primary
+          `S` - Secondary
+          `O` - Overload
+      locationId:
         type: string
-        description: Location ID of where job will be performed.
+        description: Location Id of where job will be performed.
       status:
         type: string
-        description: Job status
+        description: Job status.
         enum:
           - Active
           - Leave without pay but with benefits
           - Leave without pay and benefits
           - Leave with full pay and benefits
           - Leave with partial pay and benefits
           - Terminated
-        example: Active
       description:
         type: string
-        description: Description of job
+        description: Description of job.
       personnelChangeDate:
         type: string
         format: date
         description: Date the last change was made to the job record.
-        example: '2018-12-31'
+        example: 1970-1-1
       changeReasonCode:
         type: string
         description: The reason the last change was made to the job record.
       fullTimeEquivalency:
         type: number
         format: float
-        description: Full time eqivalency where full time = 1
-        example: 0.49
+        description: Full time equivalency where full time = 1.
+        example: 0.25
       appointmentPercent:
         type: number
         format: float
         description: FTE presented as a whole percentage.
-        example: 49
+        example: 25
       salaryStep:
         type: string
         format: integer
-        example: '1'
+        example: 1
       salaryGroupCode:
         type: string
+        description: Salary group code for the job.
+        example: 12345A
       strsAssignmentCode:
         type: string
-      supervisorOsuID:
+        description: STRS assignment code for the job.
+        example: 12
+      supervisorOsuId:
         type: string
         description: OSU ID of the supervisor for the job.
-      supervisorLastName:
-        type: string
-        description: Last name of the supervisor for the job.
       supervisorFirstName:
         type: string
         description: First name of the supervisor for the job.
+        example: John
+      supervisorLastName:
+        type: string
+        description: Last name of the supervisor for the job.
+        example: Smith
       supervisorEmail:
         type: string
         description: The primary email of the supervisor for the job.
+        example: john.smith@example.com
       supervisorPositionNumber:
         type: string
-        description: Position number of the supervisor.
+        description: Position number for the supervisor.
+        example: C12345
       supervisorSuffix:
```

Comment:
What do you think if we move these supervisor fileds under a supdervisor object? Will that be more clear?

Reply:
I like that idea